tests/models/test_horovod.py

```diff
@@ -358,7 +359,7 @@ def _compute_batch():
 #             limit_train_batches=0.2,
 #             distributed_backend='horovod'
 #         )
-#         results = trainer.fit(model)
+#         trainer.fit(model)
```

Comment:
Either undo this change or fix the assert right below

Commented due to flaky test?

Reply:
no idea, ask the author who commented it out :P